frontend/wasm/src/editor_api.rs

```diff
@@ -285,6 +285,13 @@ impl JsEditorHandle {
 		Ok(())
 	}
 
+	/// Update the value of a given UI widget, and commit it to the history
+	#[wasm_bindgen(js_name = widgetValueRevert)]
+	pub fn widget_value_revert(&self) -> Result<(), JsValue> {
+		self.dispatch(DocumentMessage::Undo);
```

Comment:
This makes me nervous. But it shouldn't be necessary anymore once you make the change I suggested above.